### ISSUE TITLE
Modal dialog: scroll-area without backdrop

### ISSUE THREAD

**Issue by comerc** [twbs/bootstrap#15483]
![_2015-01-02_12_26_19_avi](https://cloud.githubusercontent.com/assets/1025241/5595116/4b4110d8-927b-11e4-82b3-74aa3bd29a33.gif)

**Comment by peterblazejewicz** [comment]
@comerc 
Have you tried to call `handleUpdate()` method on modal dialog instance after its content is modified?
See:
https://github.com/twbs/bootstrap/issues/15106
Thanks!

**Comment by comerc** [comment]
Thanks! It works.

**Comment by peterblazejewicz** [comment]
You're welcome!

**Comment by comerc** [comment]
`handleUpdate()` does not work in `shown.bs.modal` 

and no other events to perform `handleUpdate()` on show modal

**Comment by comerc** [comment]
Oh-oh-oh, it should be inserted in many places, I believe there is a better way. 😇

**Comment by neurosnap** [comment]
version 3.1.1 doesn't have this issue, upon upgrading to 3.3.1 we noticed this issue.  It looks like it's because the modal-backdrop is being prepended .modal instead of appending to body.

v3.3.1:
https://github.com/twbs/bootstrap/blob/401889a8dc7d530d9be3b6b005afe03856089437/js/modal.js#L183

v3.1.1:
https://github.com/twbs/bootstrap/blob/a365d8689c3f3cee7f1acf86b61270ecca8e106d/js/modal.js#L156

**Comment by comerc** [comment]
I checked on the new version - now worse. Some times it works, but if not works then `handleUpdate()` does not always work now.

**Comment by bobvandevijver** [comment]
Try the following: 
```$('#myModal').data('bs.modal').handleUpdate();```

It might be better to couple it to an event, to ensure an updated DOM, but I'm not sure about that.

**Comment by comerc** [comment]
I found [jQuery Observer](http://plugins.jquery.com/tag/observer/). We may observe height .modal-backdrop. (I have not tried it, it's just an idea).

**Comment by comerc** [comment]
Also I have this bug for modal children (selectize.js for example).

**Comment by kumold** [comment]
easiest way to fix  change css of backdrop to position fixed bottom 0.

**Comment by comerc** [comment]
> easiest way to fix is changing css of backdrop to position fixed bottom 0.

Example, please.

**Comment by kumold** [comment]
http://jsbin.com/zakoji/1/

**Comment by twbs-lmvtfy** [comment]
Hi @kumold!

You appear to have posted a live example (http://jsbin.com/zakoji/1/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness
* line 12, column 3: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by kumold** [comment]
http://jsbin.com/zakoji/1/

**Comment by twbs-lmvtfy** [comment]
Hi @kumold!

You appear to have posted a live example (http://jsbin.com/zakoji/1/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by comerc** [comment]
It works! Thanks. @kumold,  you are genius! 😍

**Comment by peterblazejewicz** [comment]
Does it fix one problem introducing another? I have impression that scrolling is now affected (visible in desktop resolution), try to scroll the modal and entire screen content over backdrop.

**Comment by kumold** [comment]
Edited my example. Now it works.

**Comment by hnrch02** [comment]
Duplicate of #15136.
